docs/instrumentation-list.yaml

```diff
@@ -2836,24 +2836,6 @@ libraries:
     target_versions:
       javaagent:
       - io.projectreactor.netty:reactor-netty:[0.8.2.RELEASE,1.0.0)
-    telemetry:
-    - when: default
-      metrics:
-      - name: http.client.request.duration
```

Comment:
the two metrics removed in this PR were both from `io.opentelemetry.netty-4.1` instead of these other instrumentations